lib/Core/MemoryManager.cpp

```diff
@@ -134,11 +134,29 @@ size_t MmapAllocation::getUsedSize() const {
     return (unsigned char *)nextFreeSlot - (unsigned char *)data;
 }
 
+size_t AllocatorMap::getAppropriateBlockSize(size_t allocationSize) const {
+  return BASE_BLOCK_SIZE * (allocationSize/BASE_BLOCK_SIZE + 1);
+}
+
+MmapAllocator &AllocatorMap::getOrCreateAllocator(size_t blockSize) {
+  auto it = allocators.find(blockSize);
+  if (it == allocators.end()) {
+    it = allocators.emplace_hint(it, blockSize, flags | MAP_32BIT);
```

Comment:
Why is the MAP_32BIT hardcoded here? Wouldn't it be better to put MAP_32BIT into flags in ctor of lowmem? It is more flexible for future changes.

Reply:
I'll change it. I decided to hardcode it because it was hardcoded here as well: https://github.com/staticafi/klee/blob/236d99208f15e125aa89008c9652965ad89d3005/lib/Core/MemoryManager.cpp#L141